tools/Manual/main.cpp

```diff
@@ -70,13 +70,13 @@ void test_gmm(const string& fn_in, const string& fn_out,
   cout << "err: " << err << endl;
 
 #ifdef DO_CPP
-  string name("manual");
+  string name("Manual");
 #elif defined DO_EIGEN
   string name("manual_eigen");
```

Comment:
While we're here, let's make these consistent, and remove the special-casing in run-all.

Reply:
I agree.  I suggest keeping everything capitalised, to match the directory names in `tools`.  I'll update the pull request.

Reply:
It's a bit unclear what the intended capitalisation for "eigen" and such is.  I'll go with capitalising these as well, as that is what is done by the tool options in run-all.ps1.

Reply:
Great, thanks.   Feel free to do what feels cleanest.